packages/dap/src/client.ts

```diff
@@ -125,11 +124,11 @@ function vdafFromSpec<Spec extends KnownVdafSpec>(
 
 /**
    A client for interacting with DAP servers, as specified by
-   [draft-ietf-ppm-dap-00](https://datatracker.ietf.org/doc/html/draft-ietf-ppm-dap). Instances
+   [draft-ietf-ppm-dap-02](https://datatracker.ietf.org/doc/html/draft-ietf-ppm-dap). Instances
```

Comment:
Suggestion: Link to specific version

```suggestion
   [draft-ietf-ppm-dap-02](https://datatracker.ietf.org/doc/html/draft-ietf-ppm-dap-02). Instances
```